src/ReverseProxy/Service/RuntimeModel/Transforms/QueryTransformContext.cs

```diff
@@ -36,18 +36,18 @@ public QueryString QueryString
                     return _originalQueryString;
                 }
 
-#if NETCOREAPP3_1
-                var queryBuilder = new QueryBuilder(_modifiedQueryParameters.Select(pair => new System.Collections.Generic.KeyValuePair<string, string>(pair.Key, pair.Value.ToString())));
-#elif NETCOREAPP5_0
+#if NET5_0
```

Comment:
You probably need to use both defines until we update the SDK.
```suggestion
#if NET5_0 || NETCOREAPP5_0
```

Reply:
Yup, that makes sense. And it seems to fix the CI build.